.gitignore

```diff
@@ -7,8 +7,9 @@ reports/*
 !reports/.keep
 log
 cypress/screenshots
+cypress/results
```

Comment:
Should this be reports instead of results?

Reply:
Good point! I think it was meant to be `cypress/results` and the config file is wrong, I'll update it.